model/Human-GEM.yml

```
@@ -68370,7 +68370,7 @@
       - gene_reaction_rule: "ENSG00000131482 or ENSG00000141349 or ENSG00000152254"
       - rxnNotes: ""
       - rxnFrom: "HMRdatabase"
-      - eccodes: "3.1.3.9"
+      - eccodes: "3.1.3.9; 3.1.3.58"
```

```suggestion
      - eccodes: "3.1.3.9"
```
[EC3.1.3.9](https://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-3.1.3.9) is the canonical glucose 6-phosphatase activity, that this reactions also represents. [EC3.1.3.58](https://www.brenda-enzymes.org/enzyme.php?ecno=3.1.3.58) is a less specific sugar-phosphohydrolase found in some lactic acid bacteria.